lilypad/prompts.py

```diff
@@ -12,14 +12,16 @@
     load_config,
 )
 from .server.client import LilypadClient
+from .server.settings import get_settings
 
 _P = ParamSpec("_P")
 _R = TypeVar("_R")
 
 config = load_config()
+settings = get_settings()
 
 lilypad_client = LilypadClient(
```

Comment:
nit: seems to me like we should add `api` to the `base_url` inside of `LilypadClient` and have `settings.base_url`?

Reply:
updated to base_url